docs/source/tutorials/GEDIL4B/GEDI04_L4B documentation.ipynb

```diff
@@ -0,0 +1,298 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    #printing the shape of data to verify if rasterio was able to read the file properly

&gt;&gt;&gt;

Line #5.    # Printing the shape of data to verify if rasterio was able to read the file properly.
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>